util/conditions/v1beta2/merge_strategies.go

```diff
@@ -546,13 +552,37 @@ func sortObj(i, j string, cpMachines sets.Set[string]) bool {
 	return i < j
 }
 
+var re = regexp.MustCompile(`\s*\*\s+`)
+
 func indentIfMultiline(m string) string {
 	msg := ""
-	if strings.Contains(m, "\n") || strings.HasPrefix(m, "* ") {
+	// If it is a multiline string or if it start with a bullet, indent the message.
+	if strings.Contains(m, "\n") || re.MatchString(m) {
 		msg += "\n"
+
+		// Split the message in lines, and add a prefix; prefix can be
+		// "  " when indenting a line starting in a bullet
+		// "  * " when indenting a line starting without a bullet (indent + add a bullet)
+		// "    " when indenting a line starting with a bullet, but other lines required adding a bullet
 		lines := strings.Split(m, "\n")
+		prefix := "  "
+		hasLinesWithoutBullet := false
+		for i := range lines {
+			if !re.MatchString(lines[i]) {
+				hasLinesWithoutBullet = true
+				break
+			}
+		}
```

Comment:
We need to have this loop, because we need to add to the prefix either "  " or "* " only if at least one line doesn't start with a bullet.

If not, we are going to indent twice message with lists, which is not necessary.

Examples:

Following message doesn't need additional indentation
```
* item1
* item2
```
Because the indentation a L568 will be enough to get
```
Condition 1:
  * item1
  * item2
```

Following message 
```
Some description
* item2

```
Instead needs to be further indented + add a bullet to get the following result
```
Condition 1:
  * Some description
    * item2
```

The way to achieve this is via hasLinesWithoutBullet (first message doesn't have lines without a bullet, the second message instead has one)